juan-pablo-hernandez/tarea-5/EmptySpaceIdentifier.js

```diff
@@ -0,0 +1,99 @@
+class EmptySpaceIdentifier {
+
+  bottom(puzzleNumber, matrix, xIndex, yIndex) {
+
+    if((xIndex < 3) && (matrix[xIndex + 1][yIndex] === 'empty')) {
+
+      // Update matrix
+      matrix[xIndex + 1][yIndex] = matrix[xIndex][yIndex];
+      matrix[xIndex][yIndex] = 'empty';
+  
+      // Update right number in HTML
+      for(let i = 0; i < puzzleNumbers.length; i++) {
```

Comment:
@juan-hm use moder approaches like, forEach, map or reduce and remember variable naming for clean code.